about/index.html

```diff
@@ -92,6 +92,42 @@ <h1> About OpenVDB</h1>
           </ul>
           </p>
 
+          <p>The library was first Open Sourced by the team at DreamWorks in 2013 with a strong
+          emphasis on efficient Level Set representations and tools. It has since seen many
+          additions which have broadened its scope and feature set. Some notable additions include:</p>
+
+          <p>
+          <ul class="bulleted">
+            <li><b>OpenVDB Points:</b> spatially-organized points into OpenVDB topology for fast
+            access and greater opportunity for data compression - Contributed by DNEG (2016)</li>
+            <li><b>OpenVDB AX:</b> A domain specific language for OpenVDB, providing a fast, highly
+            parallalisable and portable solutions for custom manipulation - Contributed by DNEG (2018)</li>
+            <li><b>NanoVDB:</b> A standalone static-topology implementation of OpenVDB, offering
+            support for GPUs - Contributed by NVIDIA (2020)</li>
+          </ul>
+          </p>
+
+          <p>In August 2018 the ownership of OpenVDB was moved from DreamWorks to a new
```

Comment:
replace "ownership" with "governance". Ownership is a loaded (non-legal) term - in fact DreamWorks retains the copyright :)

Reply:
Happy to change this! But my understanding was DreamWorks also signed over the copyright of the codebase? For example, all new code we contribute is not owned by DreamWorks

Reply:
that's true for new code but to the best of my understanding old code is still technically copyright'ed by DWA. Obviously this is not clear from the code headers since we opted not to include any copy-right notices, but legally I believe they still have the copyright "ownership".